Sources/TensorFlow/Operators/NN.swift

```diff
@@ -76,36 +76,6 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 }
 
-public extension Tensor where Scalar: BinaryFloatingPoint {
-    /// Computes the batch normalized tensor along the specified axis.
-    ///
-    /// Specifically, returns `(self - mu)/(var + epsilon) * gamma + beta` where
-    /// `mu` and `var` are respectively the mean and variance of `self` along
-    /// `axis`.
-    ///
-    /// - Parameters:
-    ///     - axis: The batch dimension.
-    ///     - offset: The offset, also known as beta.
-    ///     - scale: The scale, also known as gamma.
-    ///     - epsilon: A small value added to the denominator for numerical stability.
-    @inlinable
-    @differentiable(
-        wrt: (self, offset, scale),
-        vjp: _vjpBatchNormalized where Scalar: TensorFlowFloatingPoint)
-    func batchNormalized(
```

Comment:
Removing this is necessary because `Scalar` is constrained by `BinaryFloatingPoint`, not `TensorFlowFloatingPoint`.
```
/Users/danielzheng/swift-apis/Sources/TensorFlow/Operators/NN.swift:104:19: error: global function 'rsqrt' requires that 'Scalar' conform to 'TensorFlowFloatingPoint'
        let inv = rsqrt(variance + epsilon) * scale
                  ^
/Users/danielzheng/swift-apis/Sources/TensorFlow/Operators/Math.swift:543:13: note: where 'T' = 'Scalar'
public func rsqrt<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
            ^
```

This block is duplicated with `func batchNormalized` above for some reason. I took a look at `git blame` but couldn't figure out why.

Reply:
I'm very surprised this was duplicated.